src/index.ts

```diff
@@ -67,20 +89,25 @@ export class SgidClient {
    * @param scopes Array or space-separated scopes, must include openid
    * @param nonce Specify null if no nonce
    * @param redirectUri The redirect URI used in the authorization request, defaults to the one registered with the client
+   * @param codeChallenge The code challenge from the code verifier used for PKCE enhancement
    * @returns
    */
-  authorizationUrl(
-    state: string,
-    scope: string | string[] = 'myinfo.nric_number openid',
-    nonce: string | null = generators.nonce(),
-    redirectUri: string = this.getFirstRedirectUri(),
-  ): { url: string; nonce?: string } {
+  authorizationUrl({
+    state,
+    scope = DEFAULT_SCOPE,
+    nonce = generators.nonce(),
+    redirectUri = this.getFirstRedirectUri(),
```

Comment:
can we do a check if redirectUri is in this.redirectUris?

Reply:
now that we've reverted from `redirectUris` (a string array) to just `redirectUri` (a single string input), this is no longer necessary